### PR TITLE
Adding of String Comparison #24

### PR DESCRIPTION
**Pull request by Prithwi32**
-check if the string A contains the string B
-check if the string A matches the string B. 
these 2 functions implemented

### CODE REVIEW

**Comment by Prithwi32** [comment]
@Jagroop2001 Kindly check out this let me know feedback

**Comment by jagroop-dev-11** [comment]
https://github.com/Jagroop2001/validify-string/issues/24

File: package-lock.json [hunk 1]
```diff
@@ -8,6 +8,9 @@
       "name": "validify-string",
       "version": "1.0.0",
       "license": "ISC",
```
Comment (Jagroop2001) [inline package-lock.json:10]: Please remove package-lock.json file from PR

**Comment by Prithwi32** [comment]
@Jagroop2001 could you check this now whether its alright?

**Comment by jagroop-dev-11** [comment]
https://github.com/Jagroop2001/validify-string/issues/24

**Pull request merged**